docker/Dockerfile

```diff
@@ -84,7 +85,7 @@ COPY environment.yml /root/code/garage/environment.yml
 # MAKE SURE TO DELETE MJKEY.TXT SO THAT WE DON'T PUBLISH THE KEY
 # NOTE: You MUST create and delete the mjkey.txt within the same RUN command
 ARG MJKEY
-RUN echo "${MJKEY}" > /root/.mujoco/mjkey.txt && \
+RUN echo -e "${MJKEY}" > /root/.mujoco/mjkey.txt && \
```

Comment:
why is -e necessary?

Reply:
Since I'm passing the MJKEY as a build argument in docker, I have to pass it as a single line string, that is, replace the new lines and returns as \n and \r, otherwise the make recipes are ruined by expanding these characters.
By using "echo -e", the backslash characters are expanded to reconstruct mjkey.txt.